src/utils/dateUtils.ts

```diff
@@ -50,3 +55,40 @@ export const LocalTimeToJsonPlugin = (
     return this.format('YYYY-MM-DDTHH:mm:ss');
   };
 };
+
+/**
+ * Takes a CampaignActivity and returns its start date as a Date object, or
+ * null if the start date is unset.
+ * @param activity
+ * @returns Date | null
+ */
+export function getStartDate(activity: CampaignAcitivity): Date | null {
```

Comment:
Not so sure I agree that this particular function is a date utility, because it's specifically for campaign stuff.

Reply:
now i have to change it lol

Reply:
You moved `isInFuture` but I asked that you move `getStartDate()`. 😵